### ISSUE TITLE
Recommendations for Objective-C source code in EPub and PDF 

### ISSUE THREAD

**Issue by conradwt** [thoughtbot/ios-on-rails#57]
Here's are a few recommendations which works well for ebooks in regards to Objective-C source code:

EPub/PDF
- syntax highlighting and font size
  
  Note:  I tend to read EPub on my iPad and PDF on the desktop.  Thus, I would recommend taking a look the EPub and PDF for "iOS 7 Programming Cookbook" because it a good example of having high contrast between the written text and source code.

EPub (Only)
- code image
  
  Note:  This is such a great touch to displaying code especially when the code spans multiple pages on any device.  BTW, this is the same technique you're using to display screen grabs from Xcode and the iOS Simulator.  For an Objective-C code example, I would take a look at "Effective Objective-C 2.0:  52 Ways to Improve Your iOS and OSX Programs".  The added feature here is that one can 'Click here to view code image' and use the left and right arrow keys to navigate through all the code in the book.  When you're  done looking at the code, you can click "Back to the page <NN>" to go back where you left off in the book.

In short, these are just a few things that makes it easier to read Objective-C source code within an EPub and a PDF ebook.
